.coderabbit.yaml

```diff
@@ -1,212 +1,20 @@
-language: en
+language: en-US
 early_access: true
-enable_free_tier: true
 reviews:
-  enabled: true
-  profile: chill
-  instructions: >-
-    # Code Review Instructions
-
-    - Ensure the code follows best practices and coding standards.
-    - For **Python** code, follow
```

Comment:
Is there a reason why you did not replace this with links to our code review guidelines, most of which were provided in [this Slack thread](https://dimagi.slack.com/archives/C0B4C5FT7/p1734594490063449)?

Reply:
Uh...good question. I moved it to the path_instructions, but let me add it back here and see what it does. I'm not sure if it will read the links, but let's test!